datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/prometheus.yaml

```diff
@@ -0,0 +1,146 @@
+- name: prometheus_url
```

Comment:
Are `datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/prometheus.yaml` and `datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/init_config/prometheus.yaml` intended to be used for both openmetrics and prometheus integrations ?  If yes, it might be better to call them `openmetrics.yaml` since it's the format that replaces `prometheus` format.

Reply:
I named it prometheus because the integrations are more commonly referred to prometheus-based. I can also update it to openmetrics since it is the actual base check used now. Wdyt?

Reply:
Yes, I think it makes more sense to call it `openmetrics` since it's used by `openmetrics` integration.